core/commands/files/files.go

```diff
@@ -137,11 +137,11 @@ func statNode(ds dag.DAGService, fsn mfs.FSNode) (*Object, error) {
 
 var FilesCpCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "copy files into mfs",
```

Comment:
hmm yes, this is more specific than "unixfs". it's tricky, because unixfs "is the thing that represents unix and posix files on top of the raw ipfs dag" but then, unixfs "is also the thing used by `ipfs {add, cat, ls}`". 

if `ipfs {add, cat, ls}` become merely aliases into this command, then this makes sense.

it's likely that we should just label these:

```
copy files into /files
```

what we should do is sketch out how the `$reporoot` in ipfs objects will itself work. this will help us create clear naming, docs, and uses for `files/`. (the idea that all unixfs files should be part of `files/` and thus always listable there is growing on me. `ipfs add` could place things in `files/added` or something.

A stab at it, to sketch out possibilities, but this deserves discussion:

```
reporoot
├── config  -- config file
├── dag/
├── files/
├── keys/
│   ├── id
│   └── id.pub
├── names/
├── pins/
│   ├── direct/
│   ├── indirect/
│   └── recursive/
└── records/
    ├── ipns/
    ├── provides/
    └── pubsub/
```